### PR TITLE
added badges to website

### PR DESCRIPTION
**Pull request by hesirui**
added badges to website 

@markus2330 please review my pull request

### CODE REVIEW

**Comment by markus2330** [comment]
Thank you for creating the PR!

Please also add something to the release notes.

**Comment by markus2330** [comment]
jenkins build homepage please

**Comment by hesirui** [comment]
How can I add release notes? Because I cannot log into jenkins.

**Comment by markus2330** [comment]
> How can I add release notes? Because I cannot log into jenkins.

This is described in the PR template (the one shown when you created the PR). It is also shown in the log of the build server: https://build.libelektra.org/job/libelektra/job/PR-2546/2/display/redirect

The solution is: you simply modify the file doc/news/_preparation_next_release.md which is part of the repository.

**Comment by hesirui** [comment]
Thank you, I added some release notes.

File: src/tools/rest-frontend/public/pages/main/website/home.html [hunk 1]
```diff
@@ -136,6 +136,12 @@ <h4 class="list-group-item-heading pull-left">{{ post.title}}</h4>
...
+                <a href="https://build.libelektra.org/jenkins/job/libelektra/job/master/lastBuild"><img src="https://img.shields.io/jenkins/t/https/build.libelektra.org/jenkins/job/libelektra/job/master.svg" alt="Jenkins Build Status" /></a>
+                <a href="https://travis-ci.org/ElektraInitiative/libelektra"><img src="https://travis-ci.org/ElektraInitiative/libelektra.svg?branch=master" alt="Travis Build Status" /></a>
+                <a href="https://cirrus-ci.com/github/ElektraInitiative/libelektra"><img src="https://api.cirrus-ci.com/github/ElektraInitiative/libelektra.svg" alt="Cirrus Build Status" /></a>
+                <a href="https://coveralls.io/github/ElektraInitiative/libelektra"><img src="https://img.shields.io/coveralls/github/ElektraInitiative/libelektra.svg" alt="Coverage Status" /></a>
+                <a href="https://lgtm.com/projects/g/ElektraInitiative/libelektra/alerts"><img src="https://img.shields.io/lgtm/alerts/g/ElektraInitiative/libelektra.svg?logo=lgtm&amp;logoWidth=18" alt="Total alerts" /></a></p>
```
Comment (Namoshek) [inline src/tools/rest-frontend/public/pages/main/website/home.html]: I'd rather format the HTML similar to the remaining code than having everything on one line as the lines are getting quite long. Especially the `<p>` tag should be on it's own line.

File: doc/news/_preparation_next_release.md [hunk 2]
```diff
@@ -338,7 +338,7 @@ you up to date with the multi-language support provided by Elektra.
 The website is generated from the repository, so all information about
 plugins, bindings and tools are always up to date. Furthermore, we changed:
 
-- <<TODO>>
+- Added github build status badges to website @hesirui
```
Comment (markus2330) [inline doc/news/_preparation_next_release.md]: You need to surround your name with `_()_`, i.e. change `(hesirui)` to `_(hesirui)_`. See also what the build server says: https://build.libelektra.org/jenkins/blue/organizations/jenkins/libelektra/detail/PR-2546/3/

**Comment by markus2330** [comment]
jenkins build libelektra please

File: src/tools/rest-frontend/public/pages/main/website/home.html (hunk 1 above)
Comment (markus2330) [reply src/tools/rest-frontend/public/pages/main/website/home.html]: Thank you for your quick review!

**Comment by Namoshek** [comment]
I already restarted the build and it failed again due to the wrong changelog syntax.

**Comment by markus2330** [comment]
But the homepage build job succeeded! Currently the badges are visible: https://www.libelektra.org/home
(most likely they will be gone soon,  as master-rebuilds are triggered often)

I like the addition to the webpage very much!

**Comment by Namoshek** [comment]
My fault, cache busting with ctrl+f5 didn't work for some reason. Looks good to me as well!

**Comment by markus2330** [comment]
Thank you!

**Pull request merged**